Gruntfile.js

```diff
@@ -69,11 +69,33 @@ module.exports = (grunt) => {
     let done = this.async();
 
     const m = Metalsmith(__dirname);
```

Comment:
A couple of ordering recommendations for you to ignore.
```js
const githubRegex = /github\.com\/([^/]+)\/([^/]+)\/?$/;
const oneWeek = 7 * 24 * 60 * 60;

const plugins = require('./src/plugins.json').map(plugin => {
  const result = githubRegex.exec(plugin.repository);
  if (result) {
    const [, user, repo] = result;
    Object.assign(plugin, {
      respositoryIssues: `${plugin.repository}/issues`,
      npmUrl: `https://www.npmjs.com/package/${repo}`,
      npmDownloads: `https://img.shields.io/npm/dy/${repo}.svg?maxAge=${oneWeek}`,
      npmVersion: `https://img.shields.io/npm/v/${repo}.svg?maxAge=${oneWeek}`,
      githubStars: `https://img.shields.io/github/stars/${user}/${repo}.svg?maxAge=${oneWeek}`,
      bithoundUrl: `https://www.bithound.io/github/${user}/${repo}`,
      bithoundScore: `https://www.bithound.io/github/${user}/${repo}/badges/score.svg`,
      bithoundDependencies: `https://www.bithound.io/github/${user}/${repo}/badges/dependencies.svg`
    });
  }
  return plugin;
});

const m = Metalsmith(__dirname);
```

Reply:
👍